src/main/java/com/example/event_manager/model/BillingsSummary.java

```diff
@@ -29,7 +29,7 @@ private void calculate() {
   private void calculateCoastOfAllPaided() {
     coastOfAllPaided = billingForms
         .stream()
-        .filter(b -> b.isPaided())
+        .filter(BillingForm::isPaided)
```

Comment:
there is nothing like isPaided in english :)

Reply:
hah, i promise to hang https://www.ang.pl/gramatyka/czasowniki-verbs/czasowniki-nieregularne on my wall in dormitory xD